flink-core/src/main/java/org/apache/flink/configuration/RestartStrategyOptions.java

```diff
@@ -63,21 +63,21 @@
                                                     code("fixeddelay"),
                                                     code("fixed-delay"),
                                                     link(
-                                                            "../../ops/state/task_failure_recovery#fixed-delay-restart-strategy",
+                                                            "{{.Site.BaseURL}}docs/ops/state/task_failure_recovery#fixed-delay-restart-strategy",
                                                             "here")),
                                             text(
                                                     "%s, %s: Failure rate restart strategy. More details can be found %s.",
                                                     code("failurerate"),
                                                     code("failure-rate"),
                                                     link(
-                                                            "../../ops/state/task_failure_recovery#failure-rate-restart-strategy",
+                                                            "{{.Site.BaseURL}}docs/ops/state/task_failure_recovery#failure-rate-restart-strategy",
                                                             "here")),
                                             text(
                                                     "%s, %s: Exponential delay restart strategy. More details can be found %s.",
                                                     code("exponentialdelay"),
                                                     code("exponential-delay"),
                                                     link(
-                                                            "../../ops/state/task_failure_recovery#exponential-delay-restart-strategy",
+                                                            "{{.Site.BaseURL}}docs/ops/state/task_failure_recovery#exponential-delay-restart-strategy",
```

Comment:
maybe `{{.Site.BaseURL}}/docs` to be safer?

Reply:
> maybe `{{.Site.BaseURL}}/docs` to be safer?

Yes, it will make sure the URL won't be broken if `{{.Site.BaseURL}}` is not ended with `/`. But for now it will make the URL as `https://ci.apache.org/projects/flink/flink-docs-master//docs/ops/state/task_failure_recovery/`, would it be okay?

Reply:
I took another look and can find {{.Site.BaseURL}}  sometimes used with trailing '/' and sometimes not. I think at the moment we can keep the change as is.

Reply:
After merging this PR, the link becomes "https://ci.apache.org/projects/flink/flink-docs-masterdocs/ops/state/task_failure_recovery#fixed-delay-restart-strategy" and still breaks.
I think "flink-docs-masterdocs" is not expected and a `/` is still needed.